tdrs-frontend/nginx/local/locations.conf

```diff
@@ -86,6 +87,13 @@ location = / {
     try_files $uri $uri/ /index.html;
 }
 
+proxy_intercept_errors on;
 error_page 500 502 504 /50x.html;
+location = /50x.html {
+        sub_filter '##STATUS##'  $resp_body;
```

Comment:
The problem is in Nginx we cannot inject server response into a var. So, the only option to show response is to use Django directly